### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akn0717**
Great work!
I help make some minor edits.

### CODE REVIEW

**Comment by NickleDave** [comment]
Hi @akn0717, thank you for taking the time to make these minor edits.  

I'm happy to merge in the capitalization fixes but I prefer the sentence you changed the way it was originally written. I'll request a change.

File: README.md [hunk 1]
```diff
@@ -16,26 +16,21 @@
...
-1. make it easier for researchers studying animal vocalizations to 
+1. Make it easier for researchers studying animal vocalizations to 
 apply neural network algorithms to their data
-2. provide a common framework that will facilitate benchmarking neural 
+2. Provide a common framework that will facilitate benchmarking neural 
 network algorithms on tasks related to animal vocalizations
 
-Currently the main use is automated **annotation** of vocalizations and other animal sounds.
-By **annotation**, we mean something like the example of annotated birdsong shown below:  
+Currently, the main use is an automatic annotation of vocalizations and other animal sounds. By annotation, we mean something like the example of annotated birdsong shown below:
```
Comment (NickleDave) [inline README.md:27]: Please put "annotation" in italics using single asterisks. The goal here is to define the technical meaning of the word.

See here
https://blogs.millersville.edu/bduncan/italics-quotation-marks-underscore/
> Use italics for emphasis, for unfamiliar foreign words and phrases, and for technical terms followed by definitions.

I had it in bold before but actually would prefer italics now that you draw attention to it.

File: README.md [hunk 2]
```diff
@@ -16,26 +16,21 @@
...
 <p align="center">
 <img src="https://github.com/vocalpy/vak/blob/main/doc/images/annotation-example.png?raw=True" 
 alt="spectrogram of birdsong with syllables annotated" width="400">
 </p>
 
-You give `vak` training data in the form of audio or spectrogram files with annotations, 
-and then `vak` helps you train neural network models 
-and use the trained models to predict annotations for new files.
+`vak` requires audio files or spectrogram files with annotations as the training data input, and `vak` will help you train and utilize your neural networks to make predicted annotations for new files.
```
Comment (NickleDave) [inline README.md:33]: Please change this sentence back the way it was. The informal "you" here is intentional.

**Review by NickleDave — CHANGES_REQUESTED** [review]

**Comment by akn0717** [comment]
Thank you for pointing out what you prefer!
I have made the changes as you requested.

**Review by NickleDave — APPROVED** [review]
LGTM!

**Comment by NickleDave** [comment]
Looks great, thank you @akn0717

**Comment by NickleDave** [comment]
@all-contributors please add @akn0717 for docs

**Pull request merged**